src/Shared/SemanticConventions.cs

```diff
@@ -7,116 +7,23 @@ namespace OpenTelemetry.Trace;
 
 /// <summary>
 /// Constants for semantic attribute names outlined by the OpenTelemetry specifications.
-/// <see href="https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/trace.md"/> and
-/// <see href="https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md"/>.
 /// </summary>
 internal static class SemanticConventions
```

Comment:
I haven't looked into the code so maybe a dumb question - do we need this file at all?

Reply:
See https://github.com/open-telemetry/opentelemetry-dotnet/pull/5622#discussion_r1603694410